utils/scripts/load-binary.sh

```diff
@@ -170,6 +182,8 @@ elif [ "$TARGET" = "dotnet" ]; then
         ../utils/scripts/docker_base_image.sh ghcr.io/datadog/dd-trace-dotnet/dd-trace-dotnet:latest_snapshot .
     elif [ $VERSION = 'prod' ]; then
         ../utils/scripts/docker_base_image.sh ghcr.io/datadog/dd-trace-dotnet/dd-trace-dotnet:latest .
+    elif [ $VERSION = 'dev-arm64' ]; then
```

Comment:
Where does dev-arm64 run? Would it be a better solution if we created an ARM64 image from the dd-trace-dotnet repo?

Reply:
It is never run. That's only if you want to download the arm64 binaries locally, for testing.
Yes, an arm64 image of dd-trace-dotnet would be a great thing to have. But I only wanted something that worked for now.

Reply:
I've updated the PR description to make it clearer.